python/tests/test_utils.py

```diff
@@ -236,17 +236,45 @@ def test_measurement_non_corresponding_values():
 
     m.load()
 
-    assert list(m._samples)         == [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
-    assert list(m._max_deviation)   == [2, 3, 2]
-    assert list(m._reference)       == [3, 6]
-    assert list(m._plus_tolerance)  == [0.4, 0.2, 0.1, 0.3, 0.5]
-    assert list(m._minus_tolerance) == [0.5, 0.1, 0]
-
-    assert np.array_equal(m.samples        , np.empty(0))
-    assert np.array_equal(m.max_deviation  , np.empty(0))
-    assert np.array_equal(m.reference      , np.empty(0))
-    assert np.array_equal(m.plus_tolerance , np.empty(0))
-    assert np.array_equal(m.minus_tolerance, np.empty(0))
+    msg  = 'Calibration-measurement(None):'
+    msg += 'cannot reshape array of size {} into shape 4'
+    with pybind.raises(ValueError, match=msg.format(11)):
+        _ = m._samples
+    with pybind.raises(ValueError, match=msg.format(3)):
+        _ = m._max_deviation
+    with pybind.raises(ValueError, match=msg.format(2)):
+        _ = m._reference
+    with pybind.raises(ValueError, match=msg.format(5)):
+        _ = m._plus_tolerance
+    with pybind.raises(ValueError, match=msg.format(3)):
+        _ = m._minus_tolerance
+
+def test_measurement_more_than_one_sample():
+    m = dlisio.plumbing.Measurement()
+    m.attic = {
+        'MEASUREMENT'       : [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
+        'MAXIMUM-DEVIATION' : [2, 3, 2, 2, 1, 1, 2, 2],
+        'STANDARD-DEVIATION': [3, 2, 1, 3, 4, 5, 1, 4],
+        'REFERENCE'         : [3, 6, 3, 4, 5, 1, 2, 4],
+        'PLUS-TOLERANCE'    : [0.4, 0.2, 0.1, 0.3, 0.5, 0.6, 0.5, 0.7],
+        'MINUS-TOLERANCE'   : [0.5, 0.1, 0, 1.2, 1.3, 1.6, 2.0, 1.0],
+        'DIMENSION'         : [4]
+    }
+
+    m.load()
+
+    msg  = 'Calibration-measurement(None):'
+    msg += 'cannot reshape array of size {} into shape 4'
```

Comment:
StandardDeviation + looks like sizes do not correspond.
And there should be no error from samples